server/webhook_worker.go

```diff
@@ -48,8 +50,46 @@ func (ww webhookWorker) process(msg *webhookMessage) (err error) {
 	if err = v.JiraWebhook.expandIssue(ww.p, msg.InstanceID); err != nil {
 		return err
 	}
+	// We will only send webhook events if we have a connected instance.
+	instance, err := ww.p.instanceStore.LoadInstance(msg.InstanceID)
+	if err != nil {
+		// This isn't an internal server error. There's just no instance installed.
+		return err
+	}
+
+	comments := v.JiraWebhook.Issue.Fields.Comments.Comments
+	mattermostUserID, err := ww.p.userStore.LoadMattermostUserID(instance.GetID(), comments[len(comments)-1].Author.AccountID)
```

Comment:
check whether the len of comments is greater than 0 or not else this can give panic